tozny/resource_realm.go

```diff
@@ -141,10 +151,25 @@ func resourceRealmCreate(ctx context.Context, d *schema.ResourceData, m interfac
 
 	mpcEnabled := d.Get("mpc_enabled").(bool)
 	tozidFederation := d.Get("tozid_federation_enabled").(bool)
-	err = toznySDK.RealmSettingsUpdate(ctx, d.Get("realm_name").(string), identityClient.RealmSettingsUpdateRequest{
+
+	settingsUpdateRequest := identityClient.RealmSettingsUpdateRequest{
 		MPCEnabled:             &mpcEnabled,
 		TozIDFederationEnabled: &tozidFederation,
-	})
+	}
+	customForgetPasswordLink, exists := d.GetOk("forgot_password_custom_link")
+
+	if exists {
+		customForgetPasswordLinkStr := customForgetPasswordLink.(string)
+		settingsUpdateRequest.ForgotPasswordCustomLink = &customForgetPasswordLinkStr
+	}
+	customForgetPasswordText := d.Get("forgot_password_custom_text")
```

Comment:
Does this need to be `customForgotPasswordText, exists := ...` ?

Reply:
haven't gotten to testing yet! but i did add it below let me add it here!

Reply:
good catch